src/app/multiple-records/multiple-records.component.spec.ts

```diff
@@ -18,7 +18,9 @@ describe('MultipleRecordsComponent', () => {
   beforeEach(async(() => {
     TestBed.configureTestingModule({
       imports: [RouterTestingModule, SharedModule],
-      providers: [TechRecordHelpersService],
+      providers: [
+        TechRecordHelpersService
```

Comment:
Opportunity to remove this service. We are only using it in one line of code found here `<div *ngIf="!techRecHelpers.isEmptyObject(vTechRecord.vrms); else hyphen">` in src\app\multiple-records\multiple-records.component.html